tools/collapse_collection/merge.xml

```diff
@@ -18,13 +18,13 @@
          #if $one_header:
            printf "$f.element_identifier\t"; tail -q -n +2 "$f";
          #else:
-           printf "$f.element_identifier\t"; cat "$f";
+           printf "$f.element_identifier\t"; awk '{ print $0 } END { if (NR == 0 && NF == 0) { print "" } }' "$f";
```

Comment:
You want an empty line if the file is blank?

Reply:
It will not be an empty line since the element_identifier is always present.  The awk command ensure there is a new line character if input dataset is empty.